### DISCUSSION TITLE
Permission Denied on Build/Deploy

### DISCUSSION THREAD

**Discussion by Eieyron** [wpengine/github-action-wpe-site-deploy#51] [Q&A]
Hello! Im currently developing a plugin and I want to automate the building and deployment process. 

I successfully setup the SSH Gateway and it's running on my local machine.
I was also able to setup the Secrets on the Repository.

Now i'm encountering this error whenever the Workflow file triggers.

![image](https://user-images.githubusercontent.com/30264012/201551825-65112067-384b-48c2-9a08-65d2727873ca.png)

This is the contents of my yml file:
![image](https://user-images.githubusercontent.com/30264012/201552066-34a1e801-d7c5-4b8b-86e7-234f8c91d6d8.png)

**Comment by yogbha** [comment]
Check your source and remote path...try starting with "wp-content/" instead of starting with /home in both source and remote path
